tests/load/locustfiles/stored.py

```diff
@@ -0,0 +1,394 @@
+        self.send_hello(ws)
+
+    def on_ws_message(self, ws: WebSocket, data: str) -> None:
+        """Called when received data from a WebSocket.
+
+        Args:
+            ws: WebSocket class object
+            data: utf-8 data received from the server
+        """
+        message: Message | None = self.recv(data)
+        if isinstance(message, HelloMessage):
+            self.uaid = message.uaid
+        elif isinstance(message, NotificationMessage):
+            self.send_ack(ws, message.channelID, message.version)
+        elif isinstance(message, RegisterMessage):
+            self.channels[message.channelID] = message.pushEndpoint
+        elif isinstance(message, UnregisterMessage):
+            del self.channels[message.channelID]
+
+    def on_ws_error(self, ws: WebSocket, error: Exception) -> None:
+        """Called when there is a WebSocket error or if an exception is raised in a WebSocket
+        callback function.
+
+        Args:
+            ws: WebSocket class object
+            error: Exception object
+        """
+        logger.error(str(error))
+
+        # WebSocket closures are expected
+        if isinstance(error, WebSocketConnectionClosedException):
+            return  # Don't send the exception to the UI it creates too much clutter
+
+        self.environment.events.user_error.fire(
+            user_instance=self.context(), exception=error, tb=error.__traceback__
+        )
+
+    def on_ws_close(
+        self, ws: WebSocket, close_status_code: int | None, close_msg: str | None
+    ) -> None:
+        """Called when closing a WebSocket.
+
+        Args:
+            ws: WebSocket class object
+            close_status_code: WebSocket close status
+            close_msg: WebSocket close message
+        """
+        if close_status_code or close_msg:
+            logger.info(f"WebSocket closed. status={close_status_code} msg={close_msg}")
+
+    @task(weight=98)
+    def send_notification(self):
+        """Sends a notification to a registered endpoint while connected to Autopush."""
+        if not self.ws or not self.channels:
+            logger.debug("Task 'send_notification' skipped.")
+            return
+
+        endpoint_url: str = random.choice(list(self.channels.values()))
+        self.post_notification(endpoint_url)
+
+    @task(weight=1)
+    def subscribe(self):
+        """Subscribes a user to an Autopush channel."""
+        if not self.ws:
+            logger.debug("Task 'subscribe' skipped.")
+            return
+
+        channel_id: str = str(uuid.uuid4())
+        self.send_register(self.ws, channel_id)
+
+    @task(weight=1)
+    def unsubscribe(self):
+        """Unsubscribes a user from an Autopush channel."""
+        if not self.ws or not self.channels:
+            logger.debug("Task 'unsubscribe' skipped.")
+            return
+
+        channel_id: str = random.choice(list(self.channels.keys()))
+        self.send_unregister(self.ws, channel_id)
+
+    @task(weight=20)
```

Comment:
This is not a requirement from Locust, but I suggest to adjust the weight values so they add up to 100. imo it's just mathematically easier to understand percentage wise.